docs/rst/knora-api-server/api_v1/create-a-mapping.rst

```diff
@@ -0,0 +1,358 @@
+.. Copyright © 2015 Lukas Rosenthaler, Benjamin Geer, Ivan Subotic,
+   Tobias Schweizer, André Kilchenmann, and André Fatton.
+
+   This file is part of Knora.
+
+   Knora is free software: you can redistribute it and/or modify
+   it under the terms of the GNU Affero General Public License as published
+   by the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+
+   Knora is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU Affero General Public License for more details.
+
+   You should have received a copy of the GNU Affero General Public
+   License along with Knora.  If not, see <http://www.gnu.org/licenses/>.
+
+.. _XML-to-standoff-mapping:
+
+XML to Standoff Mapping
+=======================
+
+.. contents:: :local:
+
+**************************
+The Knora Standard Mapping
+**************************
+
+A mapping allows for the conversion of XML to standoff representation in RDF and back. In order to create a TextValue with markup, both XML and the IRI of the mapping used to do the conversion to standoff have to be provided.
```

Comment:
> both XML and the IRI of the mapping used to do the conversion to standoff have to be provided

It's not clear what has to be provided in XML. Better:

"the text has to be provided in XML format, along with the IRI of the mapping that will be used to convert the markup to standoff"